phpcs.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for Liveblog">
+
+	<rule ref="WordPress-VIP" />
+	<rule ref="WordPress-Extra" />
+
+	<arg value="s"/>
```

Comment:
Include the progress (`p`), colors, extensions, basepath and parallel args - this plugin is PHP 5.6 and above, so PHPCS 3.x will be installed (3.3.1 currently).

Reply:
I was previously told not to use `--colors`:
https://github.com/WordPress/gutenberg/pull/1685#issuecomment-312780011

Reply:
`extensions` is already present.

Reply:
What do you want for `basepath` and `parallel`? @westonruter any thoughts here?

Reply:
Fixed based on your boilerplate. 15df6b2.

Reply:
`basepath="."` is enough to remove everything before the current directory.
`parallel="8"` is a good default, though I put it to 50 on my plugins - I've not yet encountered a drawback, though that doesn't mean there isn't one.